### DISCUSSION TITLE
Freac not opening at all on Monterey 12.6.5

### DISCUSSION THREAD

**Discussion by robertsouza2007** [enzo1982/freac#492] [Help & Support]
What version of freac is supposed to work with Mac OS Monterey? Version 1.1.7 is not opening on Monterey. I had 1.1.6 working fine.

**Comment by enzo1982** [comment]
fre:ac 1.1.7 has been tested and should work under Monterey.

Please check that you are using the macOS 11+ version of fre:ac, not the macOS 10 version. If you are using an Apple Silicon Mac, make sure that fre:ac is not running under Rosetta (_Context menu_->_Information_->_Open with Rosetta_ should be unchecked).

**Reply by robertsouza2007** [reply]
Yes, I am using the version 11+ and it does not open at all! My mac is a 2010 macbook pro, it's Intel. There is no Rosetta option. 
Couldn't it be something to do with permissions?

**Reply by robertsouza2007** [reply]
I think you got switched around version 10 with 11 on the download site. I installed the file named version 10 and that works!
